Mathlib/Analysis/Normed/Ring/SmoothingSeminorm.lean

```diff
@@ -0,0 +1,604 @@
+/-
+Copyright (c) 2024 María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: María Inés de Frutos-Fernández
+-/
+import Mathlib.Analysis.Normed.Ring.Seminorm
+import Mathlib.Analysis.SpecialFunctions.Pow.Continuity
+import Mathlib.Topology.MetricSpace.Sequences
+
+/-!
+# smoothingSeminorm
+In this file, we prove [BGR, Proposition 1.3.2/1][bosch-guntzer-remmert] : if `f` is a
+nonarchimedean seminorm on `R`, then `iInf (fun (n : pnat), (f(x^(n : ℕ)))^(1/(n : ℝ)))` is a
+power-multiplicative nonarchimedean seminorm on `R`.
+
+## Main Definitions
+* `smoothingSeminormSeq` : the `ℝ`-valued sequence sending `n` to `(f (x ^ n))^(1/n : ℝ)`.
+* `smoothingSeminorm'` : the iInf of the sequence `f(x^(n : ℕ)))^(1/(n : ℝ)`.
+* `smoothingSeminorm` : if `f 1 ≤ 1` and `f` is nonarchimedean, then `smoothingSeminorm'`
+  is a ring seminorm.
+
+## Main Results
+
+* `tendsto_smoothingSeminorm'` :if `f 1 ≤ 1`, then `smoothingSeminorm' f x` is the limit
+  of `smoothingSeminormSeq f x` as `n` tends to infinity.
+* `isNonarchimedean_smoothingSeminorm'` : if `f 1 ≤ 1` and `f` is nonarchimedean, then
+  `smoothingSeminorm'` is nonarchimedean.
+* `isPowMul_smoothingSeminorm'` : if `f 1 ≤ 1` and `f` is nonarchimedean, then
+  `smoothingSeminorm' f` is power-multiplicative.
+
+## References
+* [S. Bosch, U. Güntzer, R. Remmert, *Non-Archimedean Analysis*][bosch-guntzer-remmert]
+
+## Tags
+
+smoothingSeminorm, seminorm, nonarchimedean
+-/
+
+noncomputable section
+
+open Filter Nat Real
+
+open scoped Topology NNReal
+
+variable {R : Type*} [CommRing R] (f : RingSeminorm R)
```

Comment:
This is really minor: but what about changing `f` to something that remembers a norm, like `ν` or `μ`?

Reply:
I renamed `f` to  `μ`.